views/index.handlebars

```diff
@@ -0,0 +1,155 @@
+                <input type="radio" name="processDocument">
+                14 Oct 2005 <span class="info">i</span>
+              </label>
+            </div>
+            <label class="col-xs-3 col-md-1 col-lg-1 control-label">Validation</label>
+            <div id="validation" class="col-xs-9 col-md-3 col-lg-3 btn-group" data-toggle="buttons">
+              <label id="no-validation" class="btn btn-primary">
+                <input type="radio" name="validation">
+                None
+              </label>
+              <label id="simple-validation" class="btn btn-primary active">
+                <input type="radio" name="validation">
+                Simple
+              </label>
+              <label id="recursive" class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Recursively validate multipart documents">
+                <input type="radio" name="validation">
+                Recursive <span class="info">i</span>
+              </label>
+            </div>
+          </div>
+
+          <div class="form-group manual">
+            <div class="col-xs-offset-3 col-xs-9 col-sm-offset-1 col-sm-4">
+              <label class="control-label" data-toggle="tooltip" data-placement="top" title="On rec-track status, but the document is not really on rec-track">
+                <input type="checkbox" id="noRecTrack" name="noRecTrack">
+                <em>Not</em> on rec-track <span class="info">i</span>
+              </label>
+            </div>
+            <div class="col-xs-offset-3 col-xs-9 col-sm-offset-0 col-sm-4">
+              <label class="control-label">
+                <input type="checkbox" id="informativeOnly" name="informativeOnly">
+                Informative-only
+              </label>
+            </div>
+            <div class="col-xs-offset-3 col-xs-9 col-sm-offset-0 col-sm-3">
+              <label class="control-label" data-toggle="tooltip" data-placement="top" title="Check that the document is valid for automatic publication with Echidna">
+                <input type="checkbox" id="echidnaReady" name="echidnaReady">
+                <a href="https://github.com/w3c/echidna/wiki">Echidna</a>-ready <span class="info">i</span>
+              </label>
+            </div>
+          </div>
+
+        <div id="check2" class="form-group">
+          <div class="col-xs-12">
+            <button type="submit" class="btn btn-primary pull-right">Check</button>
+          </div>
+        </div>
+
+      </form>
+    </div>
+  </div>
+</div>
+</div>
+
+<div id="progressBar" class="row">
+  <div class="col-xs-12">
+    <div class="progress progress-striped active">
+      <div class="progress-bar" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="0" style="width: 0">
+        <span class="">x/y done</span>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="row">
+  <div id="alert" class="col-md-12">
+    <div class="alert alert-danger alert-dismissable hide">
```

Comment:
New alerts shouldn't be hidden.
Eg. validating 'http://localhost:1234' produces `[EXCEPTION] connect ECONNREFUSED 127.0.0.1:1234`. The DOM is updated but because of the class `hide`, the alert doesn't appear on the UI.